prow/cmd/deck/main.go

```diff
@@ -301,8 +306,40 @@ func main() {
 	// signal to the world that we're ready
 	health.ServeReady()
 
+	// cookie secret will be used for CSRF protection and should be exactly 32 bytes
```

Comment:
I'd add a comment here explaining that we sometimes accept different lengths for backwards compatibility reasons.

Reply:
backwards compat with what?